src/command_modules/azure-cli-monitor/azure/cli/command_modules/monitor/_help.py

```diff
@@ -215,36 +215,33 @@
             type: command
             short-summary: Create diagnostic settings for the specified resource.
             parameters:
+                - name: --name
+                  short-summary: The name of the diagnostic settings.
```

Comment:
This should be in params.py?
It doesn't show up currently.

![screen shot 2018-01-09 at 4 01 18 pm](https://user-images.githubusercontent.com/16448634/34749453-64f68c60-f556-11e7-934e-9438786e63d0.png)

Reply:
It needs to be --name -n in the help file.

Reply:
thanks. i will fix this.